### ISSUE TITLE
41714500: Missing font smoothing in Mojave reduces readability for all text

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20016]
#### Description

Area:
Accessibility

Summary:
Font smoothing / Subpixel rendering seems to have vanished from the first public Beta of Mojave. The replacement produces worse text rendering on non-retina displays which highly reduces readability of texts and labels in the UI.

Steps to Reproduce:
Install Mojave

Expected Results:
Text should be rendered readable on all displays

Actual Results:
Text becomes jagged and less readable on non-retina displays

Version/Build:
10.14/18A314h

Configuration:
MacBook Pro (15-inch, 2017), External Display (Eizo EV2736W) via DP

-
Product Version: 10.14
Created: 2018-07-02T09:00:05.357948
Originated: 2018-07-02T00:00:00
Open Radar Link: http://www.openradar.me/41714500
